### PR TITLE
Feature/debian packaging update

### PR DESCRIPTION
**Pull request by hanecak**
During 2017 we've neglected Debian packaging. This pull-request brings that packaging back and updates it to 3.1.0 .

### CODE REVIEW

File: debian/pom.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <parent>
+        <groupId>eu.unifiedviews</groupId>
+        <artifactId>uv-core</artifactId>
+        <version>3.2.0-SNAPSHOT</version>
+	<relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <groupId>eu.unifiedviews.packages</groupId>
+    <artifactId>unifiedviews-packages</artifactId>
+    <version>3.1.0-SNAPSHOT</version>
```
Comment (tomas-knap) [inline debian/pom.xml]: I would use 3.2.0-SNAPSHOT (as this is current dev version)

File: debian/pom.xml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        <module>unifiedviews-backend-pgsql</module>
+        <module>unifiedviews-backend-shared</module>
+        <module>unifiedviews-webapp</module>
+        <module>unifiedviews-webapp-mysql</module>
+        <module>unifiedviews-webapp-pgsql</module>
+        <module>unifiedviews-webapp-shared</module>
+    </modules>
+
+    <properties>
+        <backend-version>3.1.0</backend-version>
```
Comment (tomas-knap) [inline debian/pom.xml]: Should be 3.2.0-SNAPSHOT

Comment (tomas-knap) [reply debian/pom.xml]: Also pls check in the rest of the code

**Comment by tomas-knap** [comment]
Please check the comments above + I did not tested the whole installation from the packages

**Comment by hanecak** [comment]
Changes done, waiting for CI builds (in COMSODE/EEA) and for some subsequent "free time" opportunity.

**Comment by hanecak** [comment]
CI builds passed OK. If you do not see any other deficiencies, please merge to develop.

**Comment by tomas-knap** [comment]
Done

**Pull request merged**